compute/admin_guide/install/install_defender/install_cluster_container_defender.adoc

```diff
@@ -28,24 +28,30 @@ If you must manually specify a name, deploy your Defenders from *Manage > Defend
 . Go to *Manage > Defenders > Manage*.
 
 . Click *DaemonSets*.
++
+[IMPORTANT]
+====
+If you are using ARM architecture or multiple architectures you must edit the `daemonset.yaml` configuration file to https://cloud.google.com/kubernetes-engine/docs/how-to/prepare-arm-workloads-for-deployment#node-affinity-multi-arch-arm[prepare your workloads].
```

Comment:
@rcaballeromx We need to specify that this is only for GKE, maybe we can also add the link to the required changes so that the customer will understand why.

Reply:
hi @rcaballeromx this is still relevant